tests/resources/test-pipeline-with-custom-pip-server.py

```diff
@@ -3,10 +3,12 @@
 # Edited the compiled version manually, to remove the --trusted-host flag
 # this is so we can test for tls certs validations when launcher installs packages
 @dsl.component(base_image="quay.io/opendatahub/ds-pipelines-ci-executor-image:v1.0",
-packages_to_install=['boto3'],
-pip_index_urls=['https://nginx-service.test-pypiserver.svc.cluster.local:443/simple/'])
+packages_to_install=['numpy'],
+pip_index_urls=['https://nginx-service.test-pypiserver.svc.cluster.local/simple/'],
+pip_trusted_hosts=['nginx-service.test-pypiserver.svc.cluster.local'])
```

Comment:
we don't want to mark this as trusted, we want to do a tls verification, this should be empty as per your change [here](https://github.com/kubeflow/pipelines/pull/11151#discussion_r1738959914)

Reply:
This is out of the scope of this PR.
Created a new JIRA https://issues.redhat.com/browse/RHOAIENG-13980

Reply:
how is this out of scope? the PR is explicitly removing tls verification

Reply:
Rephrased the title and the description to explicit install https://issues.redhat.com/browse/RHOAIENG-13980